packages/gateway/src/features/user/root-resolvers.ts

```diff
@@ -635,7 +635,7 @@ function createOrUpdateUserPayload(user: GQLUserInput): IUserPayload {
     name: user.name.map((name: GQLHumanNameInput) => ({
       use: name.use as string,
       family: name.familyName?.trim() as string,
-      given: (name.firstNames || '')?.trim().split(' ') as string[]
+      given: [name.firstNames?.trim() || ''] as string[]
```

Comment:
Without the split the type is not an array string, but a string. *The type is incorrect*

Reply:
`name.firstNames?.trim() || ''` it is inside an array here.